docs/how-to-display-operator-information-in-the-marketplace-ui.md

```diff
@@ -0,0 +1,72 @@
+# How to Display Operator Information in the Marketplace UI
+
+An operator's csv must contain additional information to be displayed properly within the Marketplace UI. The Marketplace expects the following information::
```

Comment:
`An operator's CSV must contain the following annotations for it to be display properly within the Marketplace UI: `.

Reply:
I will include this in the updated commit.